modules/tutorials/pages/buckets-scopes-and-collections.adoc

```diff
@@ -0,0 +1,106 @@
+:page-pagination:
+include::partial$tutorial-globals.adoc[]
+
+= Buckets, Scopes and Collections
+
+:description: pass:q[In this section, you'll learn how to logically partition your data in Couchbase using buckets, scopes and collections.]
+
+[abstract]
+{description}
+
+== So what is a Bucket exactly?
+
+If you think in database terms, a Couchbase Bucket is analogous to a database: it's the data store where you're going to store and retrieve related information about the students.
+
+
+You can click on the menu:Dashboard[Buckets] link to access the Buckets page, then click on btn:[Add Bucket]
+
+And from this dialog, you can create the `student-bucket`
+
+image::add-student-bucket.png[width="{image-width}"]
+
+Once you have entered the bucket name, press the btn:[Add Bucket] button to return to the main bucket list.
+
+
+== Scopes and Collections
+
+There's nothing stopping you from dumping every document into the bucket. But in all but the simplest cases, it's better to provide some kind of separation. between documents of different types. Couchbase has a simple hierarchy model which allows for such separation:
+
+include::partial$student-database-diagrams.adoc[tags="couchbase-hierarchy"]
+
+You're already familiar with `clusters`, `nodes` and `buckets`. Inside a bucket you can also have any number of `scopes`, and inside a `scope` you can have any number of `collections`. Scopes and collections appear to do the same thing, so we'll look at the key differences below:
+
+[horizontal]
+*scopes*:: acts as a parent to a `collection`. You can apply permissions at the scope level, which will restrict access to the collections inside it.
+*collections*:: a collection can contain a set of related documents.
+
+
+NOTE: Remember `scopes` and `collections` are optional: you can insert documents direcly into a `bucket` if you wish.
```

Comment:
Hmm, if you don't provide a `named` scope and collection, your document will still end up in a scope, it's just called `_default`. Just thought it might be confusing to say it will insert directly into the bucket as it makes me think it won't use scopes and collections entirely.

Reply:
Good point, yes